single-exhibits.php

```diff
@@ -26,8 +26,11 @@
 				<div class="article-head">
 
 					<h2><?php the_title(); ?></h2>
+					<?php if ( get_field( 'sponsored' ) ) : ?>
+						<h3 class="sub-title"><?php the_field( 'subtitle' );?></h3>
```

Comment:
I'm trying to understand why we're checking for one field (`sponsored`) and then outputting another (`subtitle`)?